packages/p/portmidi/pspec_x86_64.xml

```diff
@@ -30,20 +31,27 @@
 </Description>
         <PartOf>programming.devel</PartOf>
         <RuntimeDependencies>
-            <Dependency release="6">portmidi</Dependency>
+            <Dependency release="7">portmidi</Dependency>
         </RuntimeDependencies>
         <Files>
+            <Path fileType="header">/usr/include/pmutil.h</Path>
             <Path fileType="header">/usr/include/portmidi.h</Path>
             <Path fileType="header">/usr/include/porttime.h</Path>
+            <Path fileType="library">/usr/lib64/cmake/PortMidi/PortMidiConfig.cmake</Path>
+            <Path fileType="library">/usr/lib64/cmake/PortMidi/PortMidiConfigVersion.cmake</Path>
+            <Path fileType="library">/usr/lib64/cmake/PortMidi/PortMidiTargets-relwithdebinfo.cmake</Path>
+            <Path fileType="library">/usr/lib64/cmake/PortMidi/PortMidiTargets.cmake</Path>
+            <Path fileType="library">/usr/lib64/libportmidi.so</Path>
+            <Path fileType="data">/usr/lib64/pkgconfig/portmidi.pc</Path>
```

Comment:
Looks like they added a pkgconfig. You can change all the deps now to `pkgconfig(portmidi)`

Reply:
Can i do it without rebuild? It was a pain to build mame

Reply:
Done